builtin/logical/pki/path_fetch_issuers.go

```diff
@@ -92,7 +93,7 @@ issuer's computed CAChain field, when non-empty.`,
 		Description: `Behavior of leaf's NotAfter fields: "err" to error
 if the computed NotAfter date exceeds that of this issuer; "truncate" to
 silently truncate to that of this issuer; or "permit" to allow this
-issuance to succeed (with NotAfter exceeding that of an issuer). Note that
+        issuance to succeed (with NotAfter exceeding that of an issuer). Note that
```

Comment:
Just seems like an accidental change

```suggestion
issuance to succeed (with NotAfter exceeding that of an issuer). Note that
```